packages/control/src/components/Protected/Sidebar/src/components/RoomOptions.js

```diff
@@ -7,17 +7,21 @@ const RoomOptions = ({ isClicked, position, room, baseUrl }) => {
   const room_Id = room?.room_url?.split("/")[2];
   const org = localStorage.getItem("currentWorkspace");
   const orgs = JSON.parse(sessionStorage.getItem("organisations"));
-  const member_id = orgs.filter(x => x.id == org)[0].member_id;
-
+  const [memberId, setMemberId] = useState("");
+  // const member_id = orgs ? orgs.filter(x => x.id == org)[0].member_id
   //  axios
   // .("https://api.zuri.chat/auth/login", {
 
   // })
+  useEffect(() => {
+    setMemberId(orgs && orgs.filter(x => x.id == org)[0].member_id);
```

Comment:
Using `find()` here is more performant. Also, consider the case where `org` is not found, this will throw an error, better to use optional chaining. So below is more robust code:
`setMemberId(orgs?.find(x => x.id == org)?.member_id);`